synapse/storage/databases/main/event_push_actions.py

```diff
@@ -64,6 +66,16 @@ def _deserialize_action(actions, is_highlight):
         return DEFAULT_NOTIF_ACTION
 
 
+@attr.s
+class EventPushSummary:
+    """Summary of pending event push actions for a given user in a given room."""
```

Comment:
if this is a helper for a specific method, then:

* the docstring should probably say so
* it should have a `_` prefix in its name
* it should probably go at the end of the file rather than the beginning, so it's not in your face as soon as you open the file.